__tests__/app.test.js

```diff
@@ -29,6 +29,7 @@ describe("/api/topics", () => {
         .get("/api/topics")
         .expect(200)
         .then(({ body }) => {
+          expect(body.topics.length).toBeGreaterThan(0);
```

Comment:
We know how many topics are in the test data, or you can check - as the data is reseeded before each test you should provide an exact figure here.